### PR TITLE
MC-13703: Added external documentation to knowledge base

### PR DESCRIPTION
**Pull request by GuillaumeVialle**
#### Changes made
- Added the external documentation URL to the return API call.

#### Related PRs
- Core: https://github.com/cloudops/cloudmc-core/pull/1134
- UI: https://github.com/cloudops/cloudmc-ui/pull/1261

### CODE REVIEW

**Review by evanluc — APPROVED** [review]

File: source/includes/administration/_knowledge_base.md [hunk 1]
```diff
@@ -118,6 +132,9 @@ Attributes | &nbsp;
 `categories.createdAt`<br/>*string* | The creation date.
 `categories.updatedAt`<br/>*string* | The update date.
 `categories.featured`<br/>*boolean* | If the category needs to be displayed.
+`categories.externalDocumentation`<br/> | List of URL and language object. Only one of articles or external documentation is provided for a category (example above shows both only as an example for the object structure).
```
Comment (sweetsweetstefan) [inline source/includes/administration/_knowledge_base.md]: List of URL and language objects. Only either 1) articles, or, 2) external documentation, will be provided for a category (example above shows both only as an example for the object structure).

File: source/includes/administration/_knowledge_base.md [hunk 2]
```diff
@@ -249,6 +280,9 @@ Attributes | &nbsp;
...
```
Comment (sweetsweetstefan) [inline source/includes/administration/_knowledge_base.md]: List of URL and language objects. Only either 1) articles, or, 2) external documentation, will be provided for a category (example above shows both only as an example for the object structure).

**Review by sweetsweetstefan — APPROVED** [review]
Just a suggestion on wording.

File: source/includes/administration/_knowledge_base.md (hunk 1 above)
Comment (GuillaumeVialle) [reply source/includes/administration/_knowledge_base.md]: 53c936aeb20dd7b658b0cde164b7849039f83219

File: source/includes/administration/_knowledge_base.md (hunk 2 above)
Comment (GuillaumeVialle) [reply source/includes/administration/_knowledge_base.md]: 53c936aeb20dd7b658b0cde164b7849039f83219

**Pull request merged**